lib/Doctrine/ORM/Tools/SchemaTool.php

```diff
@@ -677,12 +677,11 @@ private function gatherRelationJoinColumns(
             );
 
             if (! $definingClass) {
-                throw new ORMException(sprintf(
-                    'Column name "%s" referenced for relation from %s towards %s does not exist.',
+                throw new MissingColumnException(
```

Comment:
Why is there no factory for this exception?

Reply:
There is, but I don't seem to use it, my mistake.